spec/support/hyperv.Tests.ps1

```diff
@@ -158,3 +158,37 @@ Describe "New-KitchenVM with DisableSecureBoot" {
         }
     }
 }
+
+Describe "New-KitchenVM with StaticMacAddress" {
+    function New-VM {}
+    function Set-VM {}
+    function Set-VMMemory {}
+    function Set-VMNetworkAdapter {param ($VM, $StaticMacAddress)}
+    function Start-VM {}
+  
+    Mock New-VM
+    Mock Set-VM
+    Mock Set-VMMemory
+    Mock Set-VMNetworkAdapter
+    Mock Start-VM
+
+    Context "When StaticMacAddress is not specified" {
+        New-KitchenVM
+
+        It "Should not set the StaticMacAddress for the VM" {
```

Comment:
This test should include passing `-StaticMacAddress ""` to `New-KitchenVM` as that is what the script will do.